tests/projects/test_project.py

```diff
@@ -40,17 +40,53 @@ def context():
 
 def test_sync_functions():
     project_name = "project-name"
+    project_name1 = "project-name1"
     project = mlrun.new_project(project_name, save=False)
+    project1 = mlrun.new_project(project_name1, save=False)
     project.set_function("hub://describe", "describe")
+    describe_func = mlrun.import_function("hub://describe")
+    project1.set_function(describe_func, "describe")
     project_function_object = project.spec._function_objects
+    project1_file_path = pathlib.Path(tests.conftest.results) / "project1.yaml"
     project_file_path = pathlib.Path(tests.conftest.results) / "project.yaml"
+    print(project1.spec.functions)
```

Comment:
used for debugging and forgot to remove?

Reply:
this was mistakenly committed